segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeepertier2/LedgerData.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl.bookkeepertier2;
+
+import java.nio.ByteBuffer;
+import lombok.Data;
+import org.apache.bookkeeper.client.LedgerHandle;
+
+@Data
+public class LedgerData {
+    // Retrieved from ZK
+    private final LedgerHandle lh;
```

Comment:
please rename this to "ledgerHandle". It's really hard to tell what `getLh()` means by looking at other code.

Reply:
Renamed.